CGCharter-1727386911.html

```diff
@@ -51,7 +50,7 @@ <h1>
     </ul>
     <p>
       The Social Web Incubator Community Group (also known as SocialCG, or
-      SWICG) is the successor of the [Social Web Working Group](https://www.w3.org/wiki/Socialwg), which ran from
+      SWICG) is the successor of the <a href="https://www.w3.org/wiki/Socialwg" target="_blank" rel="noopener noreferer">Social Web Working Group</a>, which ran from
```

Comment:
```suggestion
      SWICG) is the successor of the <a href="https://www.w3.org/wiki/Socialwg">Social Web Working Group</a>, which ran from
```

I'd rather avoid the explicit target and rel values, as that is unusual for especially intra-W3C links, and certainly shouldn't be necessary.

Otherwise yes, HTML links preferred. Thank you!